charts/aws-ingress/values.yaml

```diff
@@ -31,3 +31,16 @@ ingress:
       ttl: 300
     ws:
       wsPort: 8081
+  s3minio:
+    enabled: false # set to true if you wish to use minio on AWS instead of using real S3
+    https:
+      externalPort: 443
+      sslCert: arn:aws:iam::00000-accountnumber-00000:server-certificate/example.com
+      sslPolicy: ELBSecurityPolicy-TLS-1-2-2017-01
+      hostname: <prefix>-s3-https.<domain>
+      ttl: 300
+    http:
+      s3Port: 9000
+    selector:
```

Comment:
Is there any reason to make this configurable vs. hardcoding it in the service?

Reply:
Currently, fake-aws-s3 uses `app: minio`, and minio-external uses `app: minio-external`. We don't use `app` anywhere else though (and use the `wireService` selector), so I figured some flexibility here would be good. Do you think it's worth removing the selector key from values (hardcoding to `app`) ?

Reply:
>  fake-aws-s3 uses app: minio, and minio-external uses app: minio-external

OK, yes that's what I thought the reasoning was.

>  Do you think it's worth removing the selector key from values (hardcoding to app) ?

Nope, looks good, was wondering the reasoning behind it!